src/sidebar/store/modules/drafts.ts

```diff
@@ -1,37 +1,25 @@
-import { createSelector } from 'reselect';
-
-import { createStoreModule, makeAction } from '../create-store';
-import { removeAnnotations } from './annotations';
-
-/**
- * @typedef {import('redux-thunk/extend-redux')} Dummy
```

Comment:
This is not imported here anymore. I have added it to tsconfig's `types` instead.